packages/block-library/src/gallery/index.php

```diff
@@ -76,13 +76,11 @@ function block_core_gallery_render( $attributes, $content ) {
 		}
 	}
 
-	$class   = wp_unique_id( 'wp-block-gallery-' );
-	$content = preg_replace(
-		'/' . preg_quote( 'class="', '/' ) . '/',
-		'class="' . $class . ' ',
-		$content,
-		1
-	);
+	$class     = wp_unique_id( 'wp-block-gallery-' );
```

Comment:
It looks like a similar refactoring is already included in `trunk`:

https://github.com/WordPress/gutenberg/blob/b54b0eb12bcac265590e15b70fee0af6edff9961/packages/block-library/src/gallery/index.php#L79-L82